test/sql/bind.test.lua

```diff
@@ -1,5 +1,6 @@
 netbox = require('net.box')
 test_run = require('test_run').new()
+box.execute([[SET SESSION "sql_seq_scan" = true;]])
```

Comment:
Why do you set the value to true if it's true by default anyway?
Anyway, a proper way to fix the tests would be to use the new `SEQSCAN` keyword, wouldn't it?

Reply:
Even if default value is `true`, it is not set to stone and setting it expilictly allow to not think about default value. As for SEQSCAN in all tests - why? These test were added not for testing SEQSCAN, so why not just use the option where I can?

Reply:
AFAIU the point of this new option is compatibility, that is in the long run, we might deprecate the option and require all users to use the `SEQSCAN` keyword instead. (Note that after a few releases, compat module options are marked obsolete and can't be changed anymore. Actually, if we used the compat module, I wouldn't even bother introducing a per session option at all, but I think it shouldn't hurt.) Our tests should always use the proper most up-to-date approach. Otherwise, other people might add more tests using this session option instead of the `SEQSCAN` keyword as a workaround for full scan.

I'm leaving this up to you, since you develop sql mostly by yourself. If you're still sure that using this session option in tests like that is fine, reply so in this thread and reassign the PR back to me - I'll merge it as is.

Reply:
No, the option is essential. We decided to leave users the ability to use SQL the way they used it before. It has nothing to do with compat. This was properly described in the document and I see no reason to believe that oprion can be dropped. Compat, in my opinion, is just what is not needed here, but now I had to use it anyway.

Using the option to avoid using SEQSCAN is a valid use case.